content/docs/command-reference/plots/modify.md

```diff
@@ -16,7 +16,7 @@ usage: dvc plots modify [-h] [-q | -v] [-t <name_or_path>] [-x <field>]
                         target
 
 positional arguments:
-  target                Metrics file to set properties to
+  target                Stage plot to set properties to.
```

Comment:
```suggestion
  target                Plot file to set properties for
```

Reply:
☝🏼  The description may need a similar update 👇🏼

Reply:
Done